users/service.go

```
@@ -315,11 +321,11 @@ func (svc service) UpdateClientSecret(ctx context.Context, token, oldSecret, new
 		return mgclients.Client{}, errors.Wrap(svcerr.ErrViewEntity, err)
 	}
 	if _, err := svc.IssueToken(ctx, dbClient.Credentials.Identity, oldSecret, ""); err != nil {
-		return mgclients.Client{}, errors.Wrap(svcerr.ErrIssueToken, err)
+		return mgclients.Client{}, errors.Wrap(errIssueToken, err)
```

In `svc.IssueToken` gRPC calls error is not handled. 
Instead of wrapping error here, we can wrap and return correct error from `svc.IssueToken`, since `svc.IssueToken` is present in same service layer.